dwave/optimization/generators.py

```diff
@@ -407,6 +409,243 @@ class as the decision variable being optimized, with permutations of its
     return model
 
 
+def capacitated_vehicle_routing_with_time_windows(demand: numpy.typing.ArrayLike,
+                                                  number_of_vehicles: int,
+                                                  vehicle_capacity: float,
+                                                  time_distances: typing.Optional[numpy.typing.ArrayLike] = None,
```

Comment:
It doesn't look like `time_distances` can actually be `None` in the code.

Reply:
Good catch. Actually I think none of the input parameters can be `None`.  Fixed